tools/validate_configs/os_compatibility_tests/batch-startup.yaml

```diff
@@ -0,0 +1,88 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+
+blueprint_name: test-batch-startup
+
+vars:
+  project_id:  ## Set GCP Project ID Here ##
+  deployment_name: test
+  region: us-central1
+  zone: us-central1-a
+  # image:
+  #   family: ubuntu-2004-lts
+  #   project: ubuntu-os-cloud
+  # image:
+  #   family: centos-7
+  #   project: centos-cloud
+  image:
+    family: rocky-linux-8
+    project: rocky-linux-cloud
+  # image:
+  #   family: debian-11
+  #   project: debian-cloud
+
+deployment_groups:
+- group: primary
+  modules:
+
+  ###########
+  # Network #
+  ###########
+
+  # Source is an embedded resource, denoted by "resources/*" without ./, ../, /
+  # as a prefix. To refer to a local resource, prefix with ./, ../ or /
+  # Example - ./resources/network/vpc
+  - id: network1
+    source: modules/network/vpc
+
+  ###########
+  # Startup #
+  ###########
+
+  - id: startup
+    source: modules/scripts/startup-script
+    settings:
+      install_ansible: true
+
+  ###########
+  # Storage #
+  ###########
+  - id: homefs
+    source: modules/file-system/filestore
+    use: [network1]
+    settings:
+      local_mount: /home
+
+  ###############
+  # Cloud Batch #
+  ###############
+
+  - id: batch-job
+    source: modules/scheduler/batch-job-template
+    use:
+    - network1
+    - homefs
+    - startup
+    settings:
+      runnable: "ansible --version"
+      machine_type: n2-standard-4
+
+  - id: batch-login
+    source: modules/scheduler/batch-login-node
+    use:
+    - batch-job
+    - homefs
```

Comment:
Again, do not use homefs here. Filestore is probably more tolerant to being used twice but it will get mounted automatically just by using the job.